rules/windows/defense_evasion_masquerading_werfault.toml

```
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/08/24"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious WerFault instance command line was detected, which may indicate an attempt to run unnoticed. Verify
+process details such as command line, network connections, file writes and parent process details as well.
```

```suggestion
Identifies a suspicious WerFault command line parameter, which may indicate an attempt to run unnoticed. 
```
Would move to `note`- 
> Verify process details such as command line, network connections, file writes, and parent process information.